### PR TITLE
Register admin account based on user input during installation and add some placeholders

### PR DESCRIPTION
**Pull request by filipsobol**
During installation, user is asked to provide e-mail address and password for admin account. Having pre-defined credentials might be a security threat.

### CODE REVIEW

**Comment by tnylea** [comment]
@Laran01 This is awesome! Thanks. I'll have to re-add this to the test functionality so it passes :) Thanks for the commit. I dig it 👍

**Comment by filipsobol** [comment]
Sorry for adding two more commits to this pull request - I was going to do it separately (still learning how to do pull requests 😟 ).

File: src/VoyagerCommand.php [hunk 1]
```diff
@@ -62,18 +65,31 @@ public function fire()
         Artisan::call('vendor:publish', ['--provider' => 'Intervention\Image\ImageServiceProviderLaravel5']);
 
         $this->info("Migrating the database tables into your application");
-        Artisan::call('migrate');
+        Artisan::call('migrate:refresh');
```
Comment (marktopper) [inline src/VoyagerCommand.php:68]: `migrate:refresh` does not sound like that good of an idea.
How about people installing Voyager for existing projects?

**Comment by marktopper** [comment]
Closing due to inactivity. Ping me to reopen.
Anyone else are welcome to make a pull request for this.

**Pull request closed without merging**